Header header
string background_image
uint16[] xpixels
uint16[] ypixels
uint16[] values
